zk-token-sdk/src/zk_token_elgamal/pod/range_proof.rs

```diff
@@ -1,31 +1,128 @@
+//! Plain Old Data types for range proofs.
+
 use crate::zk_token_elgamal::pod::{Pod, Zeroable};
+#[cfg(not(target_os = "solana"))]
+use crate::{
+    errors::ProofVerificationError,
+    range_proof::{self as decoded, errors::RangeProofError},
+};
 
-/// Serialization of range proofs for 64-bit numbers (for `Withdraw` instruction)
+/// The `RangeProof` type as a `Pod` restricted to proofs on 64-bit numbers.
 #[derive(Clone, Copy)]
 #[repr(transparent)]
 pub struct RangeProof64(pub [u8; 672]);
```

Comment:
If it's not too late, could we rename these to `...U64` to match the instructions and other types? (different PR)

Reply:
Yes, I was going to do that. I don't think SPL or any of the dependencies use range proof types directly, so I think it should be okay.